src/vue/modules/create-opportunity/index.vue

```diff
@@ -782,42 +854,49 @@ export default {
       this.form.information.maturityDate = ''
       this.form.information.formType = formType
     },
+    checkDefaultAssetSubtype () {
+      if (this.isBond) {
+        this.form.saleInformation.quoteAssets = this.baseAssets[0].code
+      } else {
+        this.form.saleInformation.quoteAssets = []
+      }
+    },
```

Comment:
function does not actually checks anything, it assigns `this.form.quoteAssets` actually. please rename